packages/miniflare/test/index.spec.ts

```diff
@@ -106,6 +108,50 @@ test("Miniflare: routes to multiple workers with fallback", async (t) => {
   t.is(await res.text(), "a");
 });
 
+test("Miniflare: custom service using Content-Encoding header", async (t) => {
+  const testBody = "x".repeat(100);
+  const { http } = await useServer(t, (req, res) => {
+    const testEncoding = req.headers["x-test-encoding"]?.toString();
+    const encoders = _transformsForContentEncoding(testEncoding);
+    let initialStream: Writable = res;
+    for (let i = encoders.length - 1; i >= 0; i--) {
+      encoders[i].pipe(initialStream);
+      initialStream = encoders[i];
+    }
+    res.writeHead(200, { "Content-Encoding": testEncoding });
+    initialStream.write(testBody);
+    initialStream.end();
+  });
+  const mf = new Miniflare({
+    script: `addEventListener("fetch", (event) => {
+      event.respondWith(CUSTOM.fetch(event.request));
+    })`,
+    serviceBindings: {
+      CUSTOM(request) {
+        return fetch(http, request);
+      },
+    },
+  });
+  t.teardown(() => mf.dispose());
+
+  const test = async (encoding: string) => {
+    const res = await mf.dispatchFetch("http://localhost", {
+      headers: { "X-Test-Encoding": encoding },
+    });
+    t.is(res.headers.get("Content-Encoding"), encoding);
+    t.is(await res.text(), testBody, encoding);
```

Comment:
Here the `encoding` variable is passed as the "message" of the `t.is()`. 
Is that what was meant?